WalletWasabi.Fluent/Models/Wallets/WalletTransactionsModel.cs

```diff
@@ -31,32 +26,12 @@ public WalletTransactionsModel(Wallet wallet)
 					  .Sample(TimeSpan.FromSeconds(1))
 					  .ObserveOn(RxApp.MainThreadScheduler)
 					  .StartWith(Unit.Default);
-
-		var transactionChanges =
-			Observable.Defer(() => BuildSummary().ToObservable())
-					  .Concat(TransactionProcessed.SelectMany(_ => BuildSummary()))
-					  .ToObservableChangeSet(x => x.GetHash());
-
-		transactionChanges.Bind(out _transactions).Subscribe();
 	}
 
-	public ReadOnlyObservableCollection<TransactionSummary> List => _transactions;
-
 	public IObservable<Unit> TransactionProcessed { get; }
 
 	public bool TryGetById(uint256 transactionId, [NotNullWhen(true)] out TransactionSummary? transactionSummary)
-	{
-		var result = List.FirstOrDefault(x => x.GetHash() == transactionId);
-
-		if (result is null)
-		{
-			transactionSummary = default;
-			return false;
-		}
-
-		transactionSummary = result;
-		return true;
-	}
+		=> _wallet.TryGetTransactionSummary(transactionId, out transactionSummary);
```

Comment:
This method (`TryGetById`) has 0 references, so `_wallet.TryGetTransactionSummary` is not used, yet.

Is there a plan to use `TryGetById` or  `TryEstimateConfirmationTime` in this class?
Will there be another component that wants to use `_wallet.TryGetTransactionSummary`?

If not, we could just remove them instead.

Reply:
Haha, I see I'm not the only one seeing 0 references to these.

Nevermind my comment.